core/iwasm/include/wasm_export.h

```diff
@@ -419,6 +419,16 @@ wasm_runtime_get_mem_alloc_info(mem_alloc_info_t *mem_alloc_info);
 WASM_RUNTIME_API_EXTERN package_type_t
 get_package_type(const uint8_t *buf, uint32_t size);
 
+/**
+ * Get the package type of a module.
+ *
+ * @param module the module
+ *
+ * @return the package type, return Package_Type_Unknown if the type is unknown
+ */
+WASM_RUNTIME_API_EXTERN package_type_t
+wasm_runtime_module_get_package_type(wasm_module_t module);
```

Comment:
How about `get_module_package_type` to be consistent with `get_package_type`?

Reply:
Hi @wenyongh, we can use that naming if you like, but it seems inconsistent with every the other API in wasm_export.h. The `get_package_type` function is the only one that doesn't start with `wasm_`. In my opinion it would be better to rename `get_package_type` in the future to be consistent with the `wasm_*` naming (if there might be an opportunity to make a breaking change like this). But in the end, it's your decision, so please confirm either way.

Reply:
OK, if you prefer to be consistent with most of the other APIs, how about using `wasm_runtime_get_module_package_type`?
And for `get_package_type`, we had better not remove it, how about adding another API `wasm_runtime_get_file_package_type` as a counterpart of it?